spiritualoutfitters.html

```diff
@@ -0,0 +1,218 @@
+---
+layout: default
+---
+
+<section class="bg-primary">
+    <div class="bg-video-player">
+        <div class="embed-responsive embed-responsive-custom">
+            <video width="1787" height="1005.1875" class="bg-jumbotron-video" autoplay="autoplay" loop="loop" muted=""
+                playsinline=""
+                src="https://videos.ctfassets.net/y3a9myzsdjan/63BuO6LHW7hpVpx3mzEvp1/dde983bb80b2efabea608f8526d14f86/spiritual-outfitters-animation1.mp4">
+                <object width="360" height="240"
+                    data="https://content.crossroads.net/framework/thirdparty/tinymce/plugins/media/moxieplayer.swf"
```

Comment:
This URL is throwing a 404. What is moxieplayer.swf?

Reply:
I'm not entirely sure on this one. AG gave me the video code snippet that was being used in another place on the site as well. I'm going to pull out that object element, it doesn't appear to be needed.